ext/nnstreamer/tensor_filter/tensor_filter_snpe_v1.cc

```diff
@@ -381,36 +381,44 @@ snpe_subplugin::configure_option (const GstTensorFilterProperties *prop)
 void
 snpe_subplugin::configure_instance (const GstTensorFilterProperties *prop)
 {
-  nns_logi ("SNPE Version: %s",
-      zdl::SNPE::SNPEFactory::getLibraryVersion ().asString ().c_str ());
+  zdl::DlSystem::Version_t ver = zdl::SNPE::SNPEFactory::getLibraryVersion ();
+
+  nns_logi ("SNPE Version: %s", ver.asString ().c_str ());
+
+  if (ver.Major != 1) {
+    nns_loge ("Invalid SNPE version, version 1.x is supported but has %d.x.", ver.Major);
+    cleanup ();
+    throw std::runtime_error ("Invalid SNPE version.");
```

Comment:
You did a good job checking ver :+1: . I think it would be better to make the error message the same. Is that okay?
```
std::string err_msg = fmt::format("Invalid SNPE version, version 1.x is supported but has {}.x.", ver.Major);
nns_loge(err_msg.c_str());
cleanup();
throw std::runtime_error(err_msg);
```

Reply:
updated. plz review the changes again.

Reply:
Thank you. Have a nice day.